### ISSUE TITLE
Example1 & Example2 are dysfunctional

### ISSUE THREAD

**Issue by gaelforget** [gaelforget/NCTiles.jl#29]
@lmilechin , I started with the example updates discussed earlier today. 

- The issue I was mentioning in `Example1.jl` (old `ex_1.jl`) is in fact three-folds. 
    - `NetCDF.close` is deprecated and I am not sure what should replace it (if anything)
    - I could not use the `write` function in the `NetCDF.jl` case
    - loading the file generated by `NetCDF.jl` using `panoply` does not offer the geo-referenced plot option (compare with the attached screen capture for the file generated by `NCDatasets.jl` backend)
- The update to `Example2.jl` (old `ex_2.jl`) revealed a problem with the `readncfile` -> `write` sequence which errors out when using the file generated by `Example1.jl`.

I just merged the updates assuming that these won't be hard to fix for you. 

Could you please take a look and send a fix via PR?

(I will wait to proceed further with updating the `ex_3` to `ex_6`)

<img width="667" alt="Screen Shot 2020-05-19 at 10 11 10 PM" src="https://user-images.githubusercontent.com/20276764/82397390-5db41200-9a1e-11ea-9e68-86fcdc73a90e.png">

**Comment by lmilechin** [comment]
I took a look at what `NetCDF` is doing now, it looks like you just don't have to call `close` as of `v0.10`. Since we still say we are compatible with `v0.9`, what I've added this morning (locally) is a check for the current version, and only call `close()` if it's before `v0.10`.

I'll continue to look into the `NetCDF` backend and make changes where needed, starting with `Example1`.

I believe I found the error you did yesterday with `Example2` as well, after I updated `NCDatasets.jl` (Was it something like `ERROR: NetCDF error: NetCDF: Access failure (NetCDF error code: -77)`?). It looks like they made a slight change in `v0.10` with how data is accessed. I've checked that the fix also works with `v0.9.5`, so it'll be in the next PR.
